registration-processor/core-processor/registration-processor-manual-verification-stage/pom.xml

```diff
@@ -125,6 +125,10 @@
 			<artifactId>micrometer-registry-prometheus</artifactId>
 			<version>${io.micrometer.prometheus.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
i think jackson dependency is already added in core and should be available in this stage, kindly cross check

Reply:
Removed Dependency